src/vs/workbench/parts/terminal/electron-browser/terminalPanel.ts

```diff
@@ -84,12 +85,14 @@ export class TerminalPanel extends Panel {
 	public setVisible(visible: boolean): TPromise<void> {
 		if (visible) {
 			if (this._terminalService.terminalInstances.length > 0) {
-				this._updateConfig();
+				this._terminalService.updateConfig();
+				this._updateFont();
 				this._updateTheme();
 			} else {
 				return super.setVisible(visible).then(() => {
 					this._terminalService.createInstance();
-					this._updateConfig();
+					this._terminalService.updateConfig();
```

Comment:
This shouldn't be needed anymore as it's called within `ITerminalService.createInstance`